### PR TITLE
Add count 

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

**Comment by akharche** [comment]
Please, provide more precise commit message

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -1547,3 +1507,38 @@ def hpat_pandas_series_nunique_impl(self, dropna=True):
...
+        raise TypingError(
+            '{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
+
+    def hpat_pandas_series_count_impl(self, level=None):
+        # """
+        # Return number of non-NA/null observations in the object
+        # Returns number of unique elements in the object
+        # Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_count
+        # """
+        return numpy.count(self._data)
```
Comment (akharche) [inline hpat/datatypes/hpat_pandas_series_functions.py]: I'm afraid numpy does not support this function. Please, check it
`module 'numpy' has no attribute 'count'`

File: hpat/tests/test_series.py [hunk 2]
```diff
@@ -2099,6 +2099,48 @@ def test_series_nunique_param1_impl(S, dropna):
                     result_param1 = hpat_func_param1(S, param1)
                     self.assertEqual(result_param1, result_param1_ref)
 
+    def test_series_count(self):
```
Comment (densmirn) [inline hpat/tests/test_series.py]: The test executes  [the old functionality](https://github.com/IntelPython/hpat/blob/master/hpat/hiframes/pd_series_ext.py#L504-L506). You should comment out that to be able to to test the implementation in new style.

**Comment by shssf** [comment]
@Rubtsowa Could you please remove all code style changes which are not related to the code you provided?
Also, please squash commits.

**Pull request merged**